pkg/kubelet/prober/worker_test.go

```diff
@@ -306,6 +307,9 @@ func TestInitialDelay(t *testing.T) {
 		})
 		m.statusManager.SetPodStatus(logger, w.pod, getTestRunningStatusWithStarted(probeType != startup))
 
+		// Set initial result for test to pass.
+		resultsManager(m, probeType).Set(testContainerID, w.initialValue, w.pod)
```

Comment:
I think the behavior should remain the same when the kubelet hasn’t restarted. Why do we need to add setting the initial result in TestInitialDelay and TestFailureThreshold?